### ISSUE TITLE
Add support for 'buf format'

### ISSUE THREAD

**Issue by amckinney** [bufbuild/vscode-buf#67]
Now that `buf format` is released, we need to integrate it here.

**Comment by haunt98** [comment]
In latest version, vscode-buf auto format my .proto files. Is there any config to disable that.

**Comment by rubensf** [comment]
Forgot to tag this is as fixed by #70.

@haunt98 see this stackoverflow response: https://stackoverflow.com/questions/61827206/vscode-disable-all-auto-formatting-on-save#:~:text=Use%20vs%2Dcode%20default%20shortcut,search%20for%20save%20without%20formatting.

TL;DR: you can add this to your settings:
```
    "[proto]": {
        "editor.formatOnSave": false
    }
```
